aries-site/src/examples/templates/FilterControls/Filters.js

```diff
@@ -0,0 +1,107 @@
+// Copyright 2021 - Hewlett Packard Enterprise Company
+
+import { useContext, useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { Box, Button, Stack, Text } from 'grommet';
+import { Filter } from 'grommet-icons';
+
+import { FilterContext } from '.';
+import { FiltersLayer } from './FiltersLayer';
+
+const FilterButton = styled(Button)`
+  border: 1px solid
+    ${({ theme }) => theme.global.colors.border[theme.dark ? 'dark' : 'light']};
+  &:hover {
+    background: transparent;
+  }
+`;
+
+const ClearFiltersButton = styled(Button)`
+  margin: 0;
+  padding: 0;
+  border: 0;
+  background: transparent;
+  font-size: inherit;
+  font-weight: 500;
+  text-decoration: underline;
+  display: flex;
+  border-radius: 0;
+  &:hover {
+    background: transparent;
+  }
+`;
+
+export const Filters = () => {
+  const {
+    applyFilters,
+    data,
+    isFiltered,
+    setIsFiltered,
+    filters,
+    filtersLayer,
+    setFiltersLayer,
+    setPreviousFilters,
+    setSearchValue,
+  } = useContext(FilterContext);
+  const [filterCount, setFilterCount] = useState();
+
+  // Provide indication for the number of filters applied
+  useEffect(() => {
+    let count = 0;
+
+    if (!filtersLayer) {
```

Comment:
is this condition to make sure if doesn't update until the filtersLayer closes? if so, adding a comment would be helpful

Reply:
Adding.